src/site/styles/layout.less

```diff
@@ -83,11 +83,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
+    .action-confirmation-overlay {
+        z-index: 200;
+        background: black;
+        position: fixed;
+        opacity: .5;
+        width: 100%;
+        height: 100%;
+        bottom: 0;
+        left: 0;
+        top: 0;
+        right: 0;
+    }
+
+    .action-confirmation-container {
+        position: fixed;
+        left: 0;
+        right: 0;
+        margin: -15% auto;
+        background: white;
+        z-index: 300;
+        width: 500px;
+        height: 200px;
+        
+        .action-confirmation-text {
+            word-wrap: break-word; 
+            position: relative;
+            top: 30%;
+        }
+        
+        .action-confirmation-buttons {
+            display: table-row;
+            position: absolute;
+            bottom: 0;
+            margin-bottom: 12px;
+            input {
+                display: table-column;
```

Comment:
Using the table-based `display` values probably isn't needed here (another plus for flex).